server/graphql/general/src/queries/insurances.rs

```diff
@@ -91,16 +91,67 @@ pub struct InsuranceConnector {
 }
 
 #[derive(Union)]
-pub enum InsuranceResponse {
+pub enum InsurancesResponse {
     Response(InsuranceConnector),
 }
```

Comment:
NIT: Can we please move `InsuranceResponse` a few lines above the `insurances` query call